packages/sdk/src/internal/adapters/multipeer/rules.ts

```diff
@@ -305,8 +305,10 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
 				message: Message<Payloads.ActorCorrection>
 			) => {
 				const syncActor = session.actorSet.get(message.payload.actorId);
-				if (syncActor && ((client.authoritative && !syncActor.grabbedBy)
-					|| (syncActor.grabbedBy === client.id))) {
+				const attachment = syncActor.initialization.message.payload.actor.attachment;
```

Comment:
It is possible for syncActor to be undefined here which will cause this to throw an exception.  Getting the attachment will need to be moved into the if block below it where syncActor will already be tested for undefined.  Alternatively you can do optional chaining on the syncActor like: 
```ts
const attachment = syncActor?.initialization.message.payload.actor.attachment;
```

Reply:
Fixed. See latest commit.